manim/mobject/geometry/shape_matchers.py

```diff
@@ -43,21 +44,24 @@ def construct(self):
 
     def __init__(
         self,
-        mobject: Mobject,
+        mobject: Mobject | Sequence[Mobject],
         color: ParsableManimColor = YELLOW,
         buff: float = SMALL_BUFF,
         corner_radius: float = 0.0,
         **kwargs,
     ) -> None:
+        target = mobject
+        if not isinstance(target, Mobject):
+            target = VGroup(*mobject)
```

Comment:
Is there a reason it needs to be a `VGroup` over a normal `Group`? With `VGroup`, all arguments must be `VMobjects` whereas `Group` works with any `Mobject`.